includes/blueprints/fetch.php

```diff
@@ -10,7 +10,46 @@
 use WP_Error;
 
 /**
- * Downloads a blueprint zip file from the specified URL.
+ * Gets blueprint from either local or remote path.
+ *
+ * @param string $path The path to the file.
+ * @return string
+ */
+function get_blueprint( string $path ) {
+	if ( filter_var( $path, FILTER_VALIDATE_URL ) ) {
+		return get_remote_blueprint( $path );
+	} else {
+		return get_local_blueprint( $path );
+	}
```

Comment:
Just a minor suggestion (from my editor) to reduce the nesting complexity a little.

```suggestion
	if ( filter_var( $path, FILTER_VALIDATE_URL ) ) {
		return get_remote_blueprint( $path );
	}

	return get_local_blueprint( $path );
```